frontend/src/components/browse/BrowseContent.js

```diff
@@ -141,12 +142,40 @@ export default function BrowseContent({
     }
   };
 
+  /**
+   * For example when filtering by location="San Francisco", the url should
+   * automatically change to active filters. This enables filtered searches
+   * to persist, so that they can be easily shareable to other users.
+   *
+   * Builds a URL and updates window state. E.g. something like:
+   * http://localhost:3000/browse?&country=Austria&city=vienna&
+   */
+  const persistFiltersInURL = (activeFilters) => {
+    // Build query params, include the `?` but ignore
+    // the standalone 'ampersand' case when no active filters are present.
+    const filteredParams = buildUrlEndingFromFilters(activeFilters);
```

Comment:
We could likely refactor this function to not return an empty ampersand, but wasn't sure of its other use cases. So handling this directly.

Reply:
I don't believe we really need the empty ampersand anywhere but to be safe we'd have to test it with the other places where its used.